enclave/core/CMakeLists.txt

```diff
@@ -95,6 +95,47 @@ if (OE_SGX)
   set_source_files_properties(sgx/td_basic.c PROPERTIES COMPILE_FLAGS
                                                         -fno-stack-protector)
 
+  # Switch compiler from MSVC to Clang for compiling enclave libraries
+  if (WIN32 AND MSVC)
+    find_program(CLANG_C_COMPILER
+                 NAMES clang
+                 PATHS "C:/Program Files/LLVM/bin"
+                 NO_DEFAULT_PATH
+                 REQUIRED)
+    execute_process(
+      COMMAND ${CLANG_C_COMPILER} --version
+      OUTPUT_VARIABLE CLANG_C_COMPILER_VERSION
+      OUTPUT_STRIP_TRAILING_WHITESPACE)
+    if (CLANG_C_COMPILER_VERSION MATCHES "clang version ([0-9]+\.[0-9]+\.[0-9]+)")
```

Comment:
Would this also match unsupported clang compilers like clang 8?

Reply:
Yes, this specific part is only concerned about capturing only the version information from the clang output.